README.md

```diff
@@ -72,8 +72,8 @@ At runtime the host launches the plug-in's Activity, the plug-in's Activity retu
                         android:label="@string/plugin_name"
                         android:targetActivity=".ui.activity.PluginActivityImpl">
                     <intent-filter>
-                        <!-- For a plug-in setting, use EDIT_SETTING instead. -->
-                        <action android:name="com.twofortyfouram.locale.intent.action.EDIT_CONDITION"/>
+                        <!-- For a plug-in setting, use ACTION_EDIT_SETTING instead. -->
+                        <action android:name="com.twofortyfouram.locale.intent.ACTION_EDIT_CONDITION"/>
```

Comment:
This line introduces a regression.

The field name is "ACTION_EDIT_CONDITION" but the Intent string is "com.twofortyfouram.locale.intent.action.EDIT_CONDITION"

https://github.com/twofortyfouram/android-plugin-api-for-locale/blob/master/pluginApiLib/src/main/java/com/twofortyfouram/locale/api/Intent.java